link.x

```diff
@@ -91,3 +99,11 @@ Create a non `pub` static variable and place it in the
 ASSERT(_einterrupts - _eexceptions <= 0x3c0, "
 There can't be more than 240 interrupt handlers.
 Fix the '.rodata.interrupts' section. (cf. #[link_section])");
+
+ASSERT(_einterrupts <= _stext, "
+The '.text' section can't be placed inside '.vector_table' section.
+Set '_stext' to an adress greater than '_einterrupts'");
```

Comment:
Too bad we can't format _einterrupts here. An address like `0x0800_0200` would be more useful than  just _einterrupts.